### PR TITLE
[master] Minor logging improvements

### PR DESCRIPTION
**Pull request by dazey3**
for #720

Signed-off-by: Will Dazey <dazeydev.3@gmail.com>

### CODE REVIEW

**Comment by dazey3** [comment]
@lukasj @rfelcman I changed `connecting`, `disconnect`, & `reconnecting` to FINEST mostly because `reconnecting_to_external_connection_pool` is already set to FINEST and all 4 appear to be tracing in the same area. 
I have no problem making these all FINE (unless you think there needs to be a separation between the first 3).

**Comment by lukasj** [comment]
I'd prefer to use `FINE` and keep `reconnecting_to_external_connection_pool` as is if you really insist on changing the log level here. The thing here is that there is too much being printed out at `FINEST` already which is affecting execution (mainly slowing it down thus possibly hiding some issues) and in certain cases one is interested in getting to know if/when connection was made

File: foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/tools/schemaframework/TableCreator.java [hunk 1]
```diff
@@ -555,7 +555,7 @@ public void extendTables(final DatabaseSession session, final SchemaManager sche
                                 try {
                                     table.addFieldOnDatabase(abstractSession, fieldDef);
                                 } catch (final DatabaseException addFieldEx) {
-                                    session.getSessionLog().log(SessionLog.FINEST,  SessionLog.DDL, "table_cannot_add_field", dbField.getName(), table.getFullName(), addFieldEx.getMessage());
```
[the remark this answers is not in the document]
Comment (dazey3) [reply foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/tools/schemaframework/TableCreator.java]: fixed! Thanks!

**Comment by dazey3** [comment]
@lukasj weird... the javadoc build failed. I don't see any direct references to files I changed and the build shows many errors.

**Comment by rfelcman** [comment]
@dazey3 this is due  #673. I am working on this.

**Comment by dazey3** [comment]
@rfelcman Ok, can I just merge this then? Or would you like me to wait until you verified a fix for the issue?

**Comment by rfelcman** [comment]
JDK 11 is more strict for Javadoc quality and https://ci.eclipse.org/eclipselink/view/Current%20Jobs/job/eclipselink-nightly-master/ will fail too.

**Comment by rfelcman** [comment]
Just merge it.

**Comment by dazey3** [comment]
Thanks everyone!

**Pull request merged**